sdk/CoreWCF/Azure.CoreWCF.AzureQueueStorage/src/CoreWCF/Channels/AzureQueueStorageBinding.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Net;
+using Azure.Storage.Queues;
+using CoreWCF.Channels;
+
+namespace Azure.Storage.CoreWCF.Channels
+{
+    /// <summary>
+    /// The class storing binding information
```

Comment:
The doc comments for public members will be rendered in MS Docs.   I believe that we'll need to spend some time tuning these as customer-facing documentation.